Teams - main branch (Make PRs against this branch!) (#140)
mulesoft/docs-access-management: modules/ROOT/pages/managing-permissions.adoc

> == Assumptions

Author: ```suggestion
```

> This page assumes that you have an Organization Administrator permission in your organization, that you have been assigned as the administrator of one of the business groups of your Organization, or that you have API Version Owner permissions and that you want to manage user permissions for an API version, a business group or the entire organization.

Reviewer: Do not make this a separate topic; rework this to be the introductory text for =Manage Permissions (which is how =Managing Permissions should be retitled).

Reviewer: Do not refer to "this section" or "this page," per style ref "self reference"

> This page assumes that you have an Organization Administrator permission in your organization, that you have been assigned as the administrator of one of the business groups of your Organization, or that you have API Version Owner permissions and that you want to manage user permissions for an API version, a business group or the entire organization.

Author: ```suggestion
To manage permissions, you must have the Organization Administrator permission in your organization, have the Organization Administrator permission over one of the business groups of your organization, or have API Version Owner permissions and want to manage user permissions for an API version, a business group or the entire organization.
```

== How Permissions Work in Anypoint Platform

> In the Anypoint Platform, users belong to an organization and have a set of permissions. These permissions are assigned using xref:teams.adoc[teams], xref:roles.adoc[roles] (deprecated), or by xref:users.adoc#grant-user-permissions[granting permissions to users individually].

Reviewer: Should this be plural, because users can (do) belong to more than one org and have more than one permission?
```suggestion
In the Anypoint Platform, users belong to organizations and have sets of permissions. These permissions are assigned using xref:teams.adoc[teams] or xref:roles.adoc[roles] (deprecated), or by xref:users.adoc#grant-user-permissions[granting permissions to users individually].
```
This is probably a good place to explain what the deprecation of Roles functionality means....

The Teams feature enables you to create groups of users and then assign the members of those groups certain permissions across multiple business groups. As an Organization Administrator, you can create and modify teams and assign cascading permissions according to organizational structure.

> Each role (deprecated) contains a list of permissions that define what a user that holds that role can do with the specific resources it scopes. +

Reviewer: ```suggestion
Each role (deprecated) contains a list of permissions that define what a user with that role can do with the specific resources it scopes. +
```
"scopes"? not just "uses" or "wants to use"?
Also, note GLOBALLY the use of "role" vs. "Role"; the name "Role" should probably be followed by a noun object, such as "feature."

> Certain Roles come with a set of default permissions. As an Organization Administrator, you can create your customized roles and assign the permissions you see fit.

Reviewer: Roles? roles?

Reviewer: come with > include (translation?)

Reviewer: "see fit" idiomatic?

Author: ```suggestion
Certain roles include a set of default permissions. As an organization administrator, you can create your customized roles and assign the permissions accordingly.
```

Depending on the product, you can grant permissions directly to a specific user, without the need for roles or teams.

=== Understanding Permissions

> Depending on the amount of products you have in the Anypoint Platform, you'll see a set of default types of _permissions_ in every new organization, team, and business group when first created. There is, however, one distinction to make between the permission types:

Reviewer: ```suggestion
Depending on the number of products you have in Anypoint Platform, every new organization, team, and business group is assigned a set of default _permissions_ when first created:
```
How does the number of products affect the default permissions?

Author: ```suggestion
Depending on which products you have in Anypoint Platform, every new organization, team, and business group is assigned a set of default _permissions_ when first created:
```

. *Product Permissions*

> * Default permissions for each Anypoint Platform product (Runtime Manager, Data Gateway, and so on). They are _environment specific_ and grant you the ability to do something within a particular environment, but not to the entire organization. +

Reviewer: Based on the suggested edit to the list intro...:
```suggestion
* Default permissions for each Anypoint Platform product (Runtime Manager, Data Gateway, and so on) are _environment specific_, granting users the ability to do something within a particular environment but not within the entire organization. +
```

> Product permissions can be assigned to teams, custom roles, or to CloudHub roles.

Reviewer: ```suggestion
These default product permissions are assigned to teams, custom roles, or CloudHub roles.
```
Yikes, other uses of the already-contested deprecated "role"....

+
. *API Permissions*
+
API Permissions vary depending on the API Manager version you are using.
+
.. *Permissions in API Manager 1.x*
+

> * Default permissions for each API managed from Anypoint Platform. They can be specific for an API version, or they can be extended to all versions. +

Reviewer: ```suggestion
* Default permissions for each API managed from Anypoint Platform are either specific to an API version or extended to all versions. +
```

> You can manage user access based on a particular API version, but you cannot extend those permissions to the entire organization. To grant user permissions to all APIs, add them to teams that have the the API Versions Owner and Portals Owner permissions.

Reviewer: ```suggestion
You can manage user access based on a particular API version, but you cannot extend those permissions to the entire organization. To grant user permissions to all APIs, add them to teams that have the API Versions Owner and Portals Owner permissions.
```

* You can assign user permissions to edit or view individual API versions or API portals using the API Version Owner and the Portal Viewer permissions.
+
[NOTE]
--

> These two API permissions grant the same scopes as the API Versions Owner and Portals Viewer roles, but limited to the API and version that you defined. +

Reviewer: Is the only difference in the permissions names whether they are singular (limited by specific API and version) or plural (not limited)?

Author: ```suggestion
These two API permissions grant the same permissions as the API Versions Owner and Portals Viewer roles, but they are limited to the API and version that you defined. +
```

> For more information about these specific scopes, see xref:roles.adoc#permission-scopes-for-default-roles[Roles].

Reviewer: I'd like to see some definition or clarity around permission "scopes"...maybe just renaming to "permission scopes"?

--
.. *Permissions in API Manager 2.x*

> * Default permissions for each API managed from API Manager. Permissions are over a specific environment and a specific resource (policies, alerts, etc). If a permission to an environment is granted to a user, it gives access to that environment immediately. Example: If you want to grant permission to create and edit alerts to a user, you have to assign the Manage Alerts permission to them.

Reviewer: ```suggestion
* Default permissions for each API managed from API Manager grant permission to access a specific environment and a specific resource (policies, alerts, and so on). If a permission to an environment is granted to a user, it gives the user access to that environment immediately. For example, if you want to grant a user permission to create and edit alerts, you assign them the Manage Alerts permission.
```

> * To Create an API in Exchange, users need one of the following Exchange permissions: Exchange Admin or Exchange Contributor

Reviewer: ```suggestion
* To create an API in Exchange, users need either the Exchange Admin permission or Exchange Contributor permission.
```

Since _API versions_ and _Products deployment environments_ are grouped under organizations (and optionally under business groups too), to access them you need to have an account that owns the necessary permissions and that belongs to its corresponding organization or business group (if such resource exists).

> Roles that are assigned at the root organization level can only reference resources that are at the root organization level, and roles that belong to a business group can only reference resources within that business group.

Reviewer: ```suggestion
Roles assigned at the root organization level can reference only resources that are at the root organization level, and roles that belong to a business group can reference only resources within that business group.
```
We still using "roles" here, although deprecated?
